config/management/src/lib.rs

```diff
@@ -487,6 +555,67 @@ pub mod tests {
         };
     }
 
+    #[test]
```

Comment:
this test and the above seem somewhat duplicate.

Reply:
This test is a subset of the end-to-end test. It was useful for testing the 2nd commit, but I've removed it now.